staging/src/k8s.io/kubectl/pkg/util/fieldpath/fieldpath.go

```diff
@@ -109,3 +109,46 @@ func SplitMaybeSubscriptedPath(fieldPath string) (string, string, bool) {
 	}
 	return parts[0], parts[1], true
 }
+
+// ConvertDownwardAPIFieldLabel converts the specified downward API field label
+// and its value in the pod of the specified version to the internal version,
+// and returns the converted label and value. This function returns an error if
+// the conversion fails.
+// Copied from k8s.io/kubernetes/pkg/apis/core/pods/helpers.go#ConvertDownwardAPIFieldLabel
```

Comment:
copying this implementation into kubectl is not desired. describe should limit itself to information contained in the pod API